modules/demo-magento/bus/events/ProductDetail.ts

```diff
@@ -0,0 +1,18 @@
+export default interface ProductDetail {
+  currency: string
```

Comment:
Events should be general and use SFX data structure. So ProductDetail should contain product type returned from `useProduct`. Its up to listeners to extract necessary data.

```
import useProduct from `#ioc/composables/useProduct`

interface ProductDetail {
  product: ReturnType<typeof useProduct>
}
```